### PR TITLE
feat(view): move title (link) to start

### PR DESCRIPTION
**Pull request by stonebuzz**
When an ```ITIL Object``` is linked to another, title column on list is at end of table 

The title is a link and it's not easy to click on it

This PR : 

- Split  ```ID``` and ```Status``` column
- Move ```title``` to start

Before : 
image

After : 
image

| Q             | A
| ------------- | ---
| Bug fix?      | yes/no
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #number

### CODE REVIEW

File: src/CommonITILObject.php [hunk 1]
```diff
@@ -6092,88 +6091,113 @@ public static function showShort($id, $options = [])
             }
             echo Search::showItem($p['output_type'], $check_col, $item_num, $p['row_num'], $align);
 
-           // First column
+            // First column ID
             $first_col = sprintf(__('%1$s: %2$s'), __('ID'), $item->fields["id"]);
```
[the remark this answers is not in the document]
Comment (stonebuzz) [reply src/CommonITILObject.php]: done

**Review by orthagh — APPROVED** [review]

File: src/CommonITILObject.php [hunk 2]
```diff
@@ -6092,88 +6091,112 @@ public static function showShort($id, $options = [])
...
-            $first_col = sprintf(__('%1$s: %2$s'), __('ID'), $item->fields["id"]);
+            // First column ID
+            echo Search::showItem($p['output_type'], $item->fields["id"], $item_num, $p['row_num'], $align);
+
+            // Second column TITLE
+            $second_column = "<span class='b'>" . $item->getName() . "</span>&nbsp;";
+            if ($item->canViewItem()) {
+                $second_column  = sprintf(
+                    __('%1$s (%2$s)'),
+                    "<a id='" . $item->getType() . $item->fields["id"] . "$rand' href=\"" . $item->getLinkURL() . "\">$second_column</a>",
```
Comment (cedric-anne) [inline src/CommonITILObject.php:6102]: If `$p['output_type'] != Search::HTML_OUTPUT`, we should not have html tags like `<a>`.

Comment (stonebuzz) [reply src/CommonITILObject.php:6102]: done

File: src/CommonITILObject.php [hunk 3]
```diff
@@ -6092,88 +6091,110 @@ public static function showShort($id, $options = [])
...
+                            'applyto' => $item->getType() . $item->fields["id"] .
+                        $rand
+                        ]
+                    )
+                );
+            }
+            echo Search::showItem($p['output_type'], $second_column, $item_num, $p['row_num'], $align);
+
+            // third column
+            $third_col = sprintf(__('%1$s: %2$s'), __('ID'), $item->fields["id"]);
```
Comment (cedric-anne) [inline src/CommonITILObject.php:6124]: Variable is overriden on next lines.
```suggestion
```

**Pull request merged**